src/main/java/io/r2dbc/postgresql/PostgresqlAuthenticationFailure.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
```

Comment:
Nit: License header date reflects the inception of a file. So in this case `2019` is sufficient.

Reply:
Hey, I just copied over this header from other files.

I will update this date.